src/antidote_pb_txn.erl

```diff
@@ -71,7 +71,7 @@ process(#fpbatomicupdatetxnreq{ops = Ops}, State) ->
     end;
 
 process(#fpbsnapshotreadtxnreq{ops = Ops}, State) ->
-    lager:info("Testing txn interface.. Received snapshot read request  ~p", [Ops]),
```

Comment:
Just remove it.  One of the reasons we use source control is so you don't need to commit commented code.